modules/@apostrophecms/page/index.js

```diff
@@ -993,18 +992,18 @@ database.`);
             } else if (position === 'before') {
               rank = target.rank;
             } else if (position === 'after') {
-              if (target.type === '@apostrophecms/trash') {
-                throw self.apos.error('invalid', 'Only the trash can can be the last child of the home page.');
+              if (target.type === '@apostrophecms/archive-page') {
+                throw self.apos.error('invalid', 'Only the archive can be the last child of the home page.');
```

Comment:
The "can" still needs to be removed here.

Reply:
This one is actually correct. I fixed the other.

Reply:
You are correct.